### DISCUSSION TITLE
2.1.0 Maven release?

### DISCUSSION THREAD

**Discussion by KingMob** [resilience4j/resilience4j#1988] [General]
Hi, it doesn't appear that 2.1.0 is on Maven. My search just turns up 2.0.2. 

Has it been released, and I'm missing it?

**Comment by RobWin** [comment]
Indeed, it has been released. but seems something is wrong.
I need to check.
Thanks

**Comment by RobWin** [comment]
It should work now

**Comment by KingMob** [comment]
Great!

On Tue Jul 4, 2023, 11:21 AM GMT, Robert Winkler ***@***.***>
wrote:

It should work now

—
Reply to this email directly, view it on GitHub
<https://github.com/resilience4j/resilience4j/discussions/1988#discussioncomment-6352576>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AAHHB5IQXN57FD4MBX5KSWLXOP4B5ANCNFSM6AAAAAAZ5RFJAI>
.
You are receiving this because you authored the thread.Message ID:
***@***.***
com>
